### PR TITLE
[@types/relationship.js] Update the types declarations exported from …

### PR DESCRIPTION
**Pull request by whinc**
…module path 'relationship.js/mode'

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/relationship.js/relationship.js-tests.ts
1. types/relationship.js/package.json
1. types/relationship.js/dist/relationship-mode.min.d.ts
1. types/relationship.js/tsconfig.json

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c3be894eafbfd457164585e1bfd8c12e1828486b
</p>

File: package.json
Comment (jakebailey) [inline package.json:1]: Please revert changes to this file.

File: types/relationship.js/dist/relationship-mode.min.d.ts [hunk 1]
```diff
@@ -1,6 +1,2 @@
-export as namespace relationshipMode;
-export = relationshipMode;
```
Comment (jakebailey) [inline types/relationship.js/dist/relationship-mode.min.d.ts:2]: When I checked the file last, it was a UMD files that declared a global. Has that changed?

Comment (whinc) [reply types/relationship.js/dist/relationship-mode.min.d.ts:2]: Thank you for pointing out the problem, I have corrected it

File: package.json
Comment (jakebailey) [inline package.json:1]: Please revert this.

File: types/relationship.js/package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "projects": [
         "https://passer-by.com/relationship/"
     ],
+    "minimumTypeScriptVersion": "4.7",
     "devDependencies": {
```
Comment (jakebailey) [inline types/relationship.js/package.json:9]: This doesn't seem required to make tests pass.

File: package.json
Comment (whinc) [reply package.json:1]: done

File: types/relationship.js/package.json (hunk 2 above)
Comment (jakebailey) [reply types/relationship.js/package.json]: Ah, ok

Comment (jakebailey) [reply types/relationship.js/package.json]: Feel free to revert that then

**Review by jakebailey — APPROVED** [review]

File: types/relationship.js/dist/relationship-mode.min.d.ts [hunk 3]
```diff
@@ -1,6 +1,4 @@
...
+export as namespace relationshipModes;
```
Comment (peterblazejewicz) [inline types/relationship.js/dist/relationship-mode.min.d.ts:1]: I don't think this is correct, or maybe project site wasn't yet updated?

> ReferenceError: Can't find variable: relationshipModes

(as or 1.2.5)

Comment (whinc) [reply types/relationship.js/dist/relationship-mode.min.d.ts:1]: ok, i will check later

**Review by jakebailey — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Comment by whinc** [comment]
Ready to merge

**Comment by whinc** [comment]
> @whinc: Everything looks good here. I am ready to merge this PR (at cfdb431) on your behalf whenever you think it's ready.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! :heart:
> 
> 

Ready to merge

**Pull request merged**